project-generator/files/README.md

```diff
@@ -0,0 +1,24 @@
+## About
+This project was generated from Playground living here:
+[Project ##PROJECT-ID##](##PROJECT-LINK##)
```

Comment:
Is this properly replaced with the project name and link? Should this be a handlebars template?

Reply:
My initial idea, was that we could update those files via simple PR to the repo, instead of redeploying whole Playground. But now I'm thinking maybe this was overengeneered and can be replaced with Handlebars templates...